lib/fishjam/room.ex

```diff
@@ -401,14 +412,34 @@ defmodule Fishjam.Room do
   @impl true
   def handle_info(%EndpointMessage{endpoint_id: to, message: {:media_event, data}}, state) do
     with {:ok, peer} <- State.fetch_peer(state, to),
-         socket_pid when is_pid(socket_pid) <- Map.get(peer, :socket_pid) do
-      send(socket_pid, {:media_event, data})
+         socket_pid when is_pid(socket_pid) <- Map.get(peer, :socket_pid),
+         node_name when not is_nil(node_name) <- Map.get(peer, :node_name),
```

Comment:
Shouldn't happen in any scenario, media events comes after establishing connection. I would prefer to crash when this happens than `handle` it 🤔